src/status_im/contexts/wallet/collectible/events.cljs

```diff
@@ -131,6 +137,52 @@
               :account    current-viewing-account
               :amount     collectibles-request-batch-size}]]]})))
 
+(rf/reg-event-fx
+ :wallet/collectible-ownership-update-finished-with-error
+ (fn [{:keys [db]} [{:keys [accounts message chainId]}]]
+   (let [address   (first accounts)
+         chain-ids (get-in db [:wallet :ui :collectibles :fetching address])]
+     (log/error "[wallet] collectibles ownership update failed"
+                {:address  address
+                 :chain-id chainId
+                 :error    message})
+     {:db (cond-> db
+            (some? chain-ids)
+            (update-in [:wallet :ui :collectibles :fetching address] disj chainId))})))
+
+(rf/reg-event-fx
+ :wallet/collectible-ownership-update-finished
+ (fn [{:keys [db]} [{:keys [accounts message chainId]}]]
+   (let [address                (first accounts)
+         {:keys [added]}        (transforms/json->clj message)
```

Comment:
The message contains `added`, `updated` and `removed` keys which will have an array of maps with the contract address and token ID of the collectible. Later, we can handle the `updated` or `removed` keys and update the data in app-db for all addresses. For now, we focus on addresses which were never fetched.